core/io/resource_uid.cpp

```diff
@@ -31,9 +31,10 @@
 #include "resource_uid.h"
 
 #include "core/config/project_settings.h"
-#include "core/crypto/crypto.h"
+#include "core/crypto/crypto_core.h"
 #include "core/io/dir_access.h"
 #include "core/io/file_access.h"
+#include "core/os/os.h"
```

Comment:
I don't see what uses `OS` right away, is it actually needed, or is there a more specific header we can include instead?